validate.js

```diff
@@ -0,0 +1,51 @@
+const fsp = require("fs").promises;
```

Comment:
I completely forgot about the CLI! I'll do that now (but I think `npx` can be omitted because `./node_modules/.bin` is locally added to the `$PATH` in npm scripts).

Reply:
It never works for me, so I always include `npx` to make sure it works. If there is no harm leaving it in, do you mind leaving it in?

Reply:
Interesting — what OS are you using? (I tested on Arch Linux.) I am a bit apprehensive about using `npx` because it has a somewhat long start-up time and re-downloads the package and its dependencies, but `node ./node_modules/.bin/ajv` seems to do the job as well (even when the working directory is not the root of the repository), so I will change it to that (this should also resolve any issues that would arise from the execute permission not being set on the `ajv` binary for whatever reason, e.g. if the repository is on an NTFS partition mounted with NTFS-3G).

Reply:
I'm on MacOS. I just hate how `node ./node_modules/.bin/ajv` looks, but it does the job.